xclim/indices/_multivariate.py

```diff
@@ -210,8 +210,10 @@ def cold_and_dry_days(
     thresh = resample_doy(pr_per, pr)
     pr25 = pr < thresh
 
-    cold_and_dry = np.logical_and(tg25, pr25).resample(time=freq).sum(dim="time")
-    return to_agg_units(cold_and_dry, tas, "count")
+    cold_and_dry = cast(xarray.DataArray, np.logical_and(tg25, pr25))
```

Comment:
What's the difference between this and 
```suggestion
    cold_and_dry: xr.DataArray = np.logical_and(tg25, pr25)
```
?

Reply:
I'm glad you asked ;)

The `variable: sometype =` approach is for specifying that the outputs from an assignment that would return `Any` is constrained to a specific `sometype` instead.

The `cast(sometype, ...)` approach is used to tell the type checker that you should treat the variable as `sometype` regardless of what the type hinting from the assignment would suggest. Casting here is needed because `numpy` does not understand that if passed `xarray` objects, it should output `xarray` objects (instead, it thinks they are `ndarray`).